COMMITTERS.md

```diff
@@ -0,0 +1,90 @@
+
+**develop branch** - The branch where bug fixes against the latest release or
+release candidate are merged.
+
+##Pull request checklist
+* Add Unit tests.
+* All tests pass (see Run all tests section).
+* Check component design
+* Check amount of needed memory, memory leaks.
+* Assertion must be used in right way.
+* Are there cyclic dependencies?
+* Level of abstraction adequate?
+* Are there any platform gotchas? (Does a change make an assumption about
+   platform specific behavior that is incompatible with other platforms?  e.g.
+   Windows paths vs. POSIX paths.).
+* Thread safe code.
+* No deadlocks.
+* Doxygen API documentation available?
+* Do you have thread specific comment provided? (if another thread uses added method/classes, 
+   there must be comment which explains: When method will be called by thread? Which thread calls the method? e.g.)
+* Do you have specific realization code comments?
+* Add log messages.
+* There are no Google code style errors. (You can download Google [cppint.py](https://raw.githubusercontent.com/google/styleguide/gh-pages/cpplint/cpplint.py))
+* Check branch naming.
+* Check correct commit messages in commits (see Pull request message section).
+* Check correct pull request target (`master` or `develop`)
+* Check correct pull request message
+* Add reviewers
+
+##Run all tests
+* Run in root build directory `make test`
+
+##Branch naming
+Branch name should be:
+* `hotfix/<task name>` or
+* `feature/<task name>`
+
+##Pull request message
+* Describe the reason why you created pull request and what changes there are.
+* Add related Jira ticket as link.
+   ( EXAMPLE:
+   `Related: [APPLINK-xxxxx](put direct link here)`)
+* If there was an old pull request, add link. ( EXAMPLE: `Old pull request is [here](put direct link here)`)
+
+##Adding reviewers:
+* Add one domain expert
+* Add your mentor (for junior developers only)
+* Add all junior developers (for junior developers only)
+
+`Note`: Anyone from SDL team developers can review any pull request (“reviewed” comment is not required)
+
+##Review process
+* Reviewers can leave comments to code only in `commits` tab.
+* Contributor must answer each comment (Contributor should enter name of the person whom he addresses in review comment).
+* In case contributor is disagree with reviewer, contributor writes his opinion
+* In case contributor is agree with reviewer, contributor leaves link to new commit with changes.
+* When reviewer is agree with all changes in pull request, he must leave `Reviewed` comment.
+
+##Successfully passed review:
+* Contributor answered to all comments
+* Contributor fixed all mistakes
+* All reviewers left `Reviewed` comment
+
+##Merging
+* Rebase in case of existed conflicts.
+* Contributor can squash commits in case of adding new comments to existed comments.
+* Contributor can squash commits in case of deleting code that was added previously or adding code that was deleted previously.

```

Comment:
@OHerasym @AGaliuzov I propose to replace two lines above with something like:
`Contributor can change existing commits in case of changes in code at the same place`
to avoid misunderstandings. As for me they are too complex.